reflex/components/component.py

```diff
@@ -846,7 +875,7 @@ def _get_imports(self):
 
         return imports
 
-    def _get_custom_code(self) -> str:
+    def _get_dynamic_imports(self) -> str:
```

Comment:
Isn't this method being defined twice now (also on line 520)?

Reply:
It should be Component and NoSSRComponent(Component) so different classes.